runtime/ftplugin/man.vim

```diff
@@ -23,7 +23,7 @@ setlocal foldcolumn=0 colorcolumn=0 nolist nofoldenable
 
 setlocal tagfunc=man#goto_tag
 
-if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
+if !get(g:, 'no_plugin_maps', 0) && !get(g:, 'no_man_maps', 0)
```

Comment:
You are right. I thought it was a bug, because despite setting `g:no_man_maps = 0` the condition would evaluate to false and hence no local mappings contrary to the value of `g:no_man_maps`. I'll fix it, apparently Vim doesn't like double negatives :smile:.